linux_os/guide/system/permissions/files/permissions_important_account_files/file_permissions_etc_shadow/rule.yml

```diff
@@ -58,7 +58,7 @@ ocil: |-
 
 fixtext: '{{{ fixtext_file_permissions(file="/etc/shadow", mode=target_perms_octal) }}}'
 
-srg_requirement: '{{{ srg_requirement_file_permission(file="/etc/shadow", mode=target_perms_octal) }}}'
+srg_requirement: 'The {{{ full_name }}} /etc/shadow file must have 0000 to prevent unauthorized access.'
```

Comment:
Since Ubuntu has a different value, shouldn't the `target_perms_octal` variable be used here, instead of `0000`?

Reply:
The issue is that "or less permissive" makes no sense for 0000. Maybe we need to add a Jinja statement around this.

Reply:
I see. I think a Jinja statement would work.